infra/src/main/kotlin/fi/fta/geoviite/infra/dataImport/InfraModelMigration.kt

```diff
@@ -172,6 +172,7 @@ class V12_01__InfraModelMigration : BaseJavaMigration() {
                 val metadata = metadatas[xmlFile.nameWithoutExtension]
                 if (metadata == null) logger.warn("No metadata available for file $readablePath")
                 val (plan, file) = parseGeometryPlan(
+                    type,
```

Comment:
Näitä muutoksia levisi vähän näihin suunnitelmia käsittäviin tiedostoihin: aiemmin suunnitelman lähde tallennettiin planin kyljessä erikseen ja se näkyi vain plan-headerissa. Laitoin sen nyt osaksi itse plania jotta voi käyttää noissa element-listauksissa myös.